packages/components/src/internal/app/actions.ts

```diff
@@ -60,16 +61,17 @@ export const setReloadRequired = () => ({ type: SET_RELOAD_REQUIRED });
 
 export const doResetQueryGridState = () => ({ type: RESET_QUERY_GRID_STATE });
 
-export function menuInit(currentProductId: string, userMenuProductId: string, productIds?: List<string>) {
+export function menuInit(currentProductId: string, appProductId: string, productIds?: List<string>) {
     return (dispatch, getState) => {
         let menu = getState().routing.menu;
         if ((!menu.isLoaded && !menu.isLoading) || menu.needsReload) {
             if (!menu.needsReload) {
                 dispatch({
                     type: MENU_LOADING_START,
                     currentProductId,
-                    userMenuProductId,
-                    productIds, // when undefined, this returns all menu sections for modules in this container
+                    userMenuProductId: appProductId,
+                    // when productIds is undefined, this returns all menu sections for modules in this container, which we currently don't ever want
```

Comment:
if returning all menu sections is something we don't ever want (as the comment says), then should we make the `productIds` param required?